Formula/neovim-qt.rb

```diff
@@ -0,0 +1,79 @@
+class NeovimQt < Formula
+  desc "Neovim GUI, in Qt5"
+  homepage "https://github.com/equalsraf/neovim-qt"
+  url "https://github.com/equalsraf/neovim-qt/archive/v0.2.16.1.tar.gz"
+  sha256 "971d4597b40df2756b313afe1996f07915643e8bf10efe416b64cc337e4faf2a"
+  license "ISC"
+  head "https://github.com/equalsraf/neovim-qt.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "neovim-remote" => :test
+  depends_on "neovim"
+  depends_on "qt@5"
+
+  on_linux do
+    depends_on "gcc"
+  end
+
+  fails_with gcc: "5" # Qt needs GCC
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args, "-DUSE_SYSTEM_MSGPACK=ON"
+    system "cmake", "--build", "build"
+
+    if OS.mac?
+      prefix.install "build/bin/nvim-qt.app"
+      bin.install_symlink prefix/"nvim-qt.app/Contents/MacOS/nvim-qt"
+    else
+      bin.install "build/bin/nvim-qt"
+    end
+  end
+
+  test do
```

Comment:
```suggestion
  test do
    return if OS.linux? && ENV["HOMEBREW_GITHUB_ACTIONS"] # qt.qpa.xcb: could not connect to display
```
Let's just do this on Linux.